src/test/java/se/liu/ida/hefquin/engine/data/impl/VocabularyMappingTest.java

```diff
@@ -0,0 +1,113 @@
+package se.liu.ida.hefquin.engine.data.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.vocabulary.OWL;
+import org.apache.jena.vocabulary.RDF;
+import org.junit.Test;
+
+import se.liu.ida.hefquin.engine.data.Triple;
+import se.liu.ida.hefquin.engine.data.VocabularyMapping;
+import se.liu.ida.hefquin.engine.query.SPARQLGraphPattern;
+import se.liu.ida.hefquin.engine.query.SPARQLUnionPattern;
+import se.liu.ida.hefquin.engine.query.TriplePattern;
+import se.liu.ida.hefquin.engine.query.impl.SPARQLUnionPatternImpl;
+import se.liu.ida.hefquin.engine.query.impl.TriplePatternImpl;
+
+public class VocabularyMappingTest {
+
+	@Test
+	public void VocabularyMappingConstructorTest() {
+		final Set<Triple> testTriples = CreateTestTriples();
+		final List<Triple> triplesList = new ArrayList<Triple>();
+		triplesList.addAll(testTriples);
```

Comment:
You can add all the elements of another collection directly in the constructor. I'm not sure though why do you need to have both a set and a list.
```suggestion
		final List<Triple> triplesList = new ArrayList<Triple>(testTriples);
```